### PR TITLE
Fix bugs

### PR DESCRIPTION
**Pull request by michiomochi**
Hi, 
I fixed some bugs.
I will write fixes detail to comment in the code 📝 
please check 🙏

### CODE REVIEW

File: lib/capistrano/tasks/nodebrew.rake [hunk 1]
```diff
@@ -13,17 +13,15 @@ namespace :nodebrew do
...
         error "nodebrew: #{nodebrew_node} is not installed"
         exit 1
       end
     end
   end
 
   task :map_bins do
-    nodebrew_bin = fetch(:nodebrew_bin)
-
+    SSHKit.config.default_env.merge!({ nodebrew_root: fetch(:nodebrew_path) })
```
Comment (michiomochi) [inline lib/capistrano/tasks/nodebrew.rake:24]: The following error will occur, If there aren't `NODEBREW_ROOT` environment variable.

```
SSHKit::Command::Failed: /usr/local/nodebrew/current/bin/nodebrew ls exit status: 2
/usr/local/nodebrew/current/bin/nodebrew ls stdout: Nothing written
/usr/local/nodebrew/current/bin/nodebrew ls stderr: No such file or directory at /usr/local/nodebrew/current/bin/nodebrew line 572.
from /Users/michiomochi/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/sshkit-1.14.0/lib/sshkit/command.rb:99:in `exit_status='
```

I referred to capistrano-rbenv. 👀 
https://github.com/capistrano/rbenv/blob/master/lib/capistrano/tasks/rbenv.rake#L18

File: lib/capistrano/tasks/nodebrew.rake [hunk 2]
```diff
@@ -50,8 +48,9 @@ namespace :load do
       end
     }
 
-    set :nodebrew_bin, "#{fetch(:nodebrew_path)}/current/bin/nodebrew"
+    set :nodebrew_bin, -> { "#{fetch(:nodebrew_path)}/current/bin/nodebrew" }
```
Comment (michiomochi) [inline lib/capistrano/tasks/nodebrew.rake:51]: `nodebrew_path` is determined by `nodebrew_type` variable. 
https://github.com/kuy/capistrano-nodebrew/pull/1/files#diff-b8e85b16e6781e0a0ad23cd345030f43R44

`nodebrew_type` is input by `deploy.rb`, and this code is calling before loading `deploy.rb`.
So, `nodebrew_bin` should be lazy evaluation.

File: lib/capistrano/tasks/nodebrew.rake [hunk 3]
```diff
@@ -13,17 +13,15 @@ namespace :nodebrew do
         exit 1
       end
 
-      versions = capture("#{nodebrew_bin} ls").lines.map{|l| l.strip}.reject{|l| l.empty?}
-      unless versions.include? nodebrew_node
+      unless test "[ -d #{fetch(:nodebrew_node_dir)} ]"
```
Comment (michiomochi) [inline lib/capistrano/tasks/nodebrew.rake:16]: The following error will occur in my environment. 🤔 
```
SSHKit::Command::Failed: /usr/local/nodebrew/current/bin/nodebrew ls exit status: 2
/usr/local/nodebrew/current/bin/nodebrew ls stdout: Nothing written
/usr/local/nodebrew/current/bin/nodebrew ls stderr: No such file or directory at /usr/local/nodebrew/current/bin/nodebrew line 572.
from /Users/michiomochi/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/sshkit-1.14.0/lib/sshkit/command.rb:99:in `exit_status='
```

So, change the logic to checking installed directory of specified version  exist.
The logic is also adopted in capistrano-rbenv.
see: https://github.com/capistrano/rbenv/blob/master/lib/capistrano/tasks/rbenv.rake#L10

**Comment by michiomochi** [comment]
@kuy please review it 🙏

**Comment by michiomochi** [comment]
@kuy 
Thank you for your quick reaction! 😊 
Please release  🙏 😉

**Comment by kuy** [comment]
@michiomochi Nice works, thanks a lot!! Just released [0.0.3](https://rubygems.org/gems/capistrano-nodebrew/versions/0.0.3)! But I didn't test this version enough. Please let me know or send PRs if you get problems 😉

**Comment by michiomochi** [comment]
寿司ゆき:thanks

**Pull request merged**